### ISSUE TITLE
SVGs not showing at all Google Pixel 4a 5G - Android 11

### ISSUE THREAD

**Issue by bracketsberlin** [muak/SvgImageSource#20]
Hi,
unfortunately, none of the SVGs are showing up on my Google Pixel 4a 5G (Android 11). When testing with an Android 11 emulator, the svgs are showing up though.

Do you have any hints how to debug/test this weird behaviour?

Thanks

**Comment by mroseteo** [comment]
@bracketsberlin did you find a solution? Am having the same issue with some 20:9 aspect ratio devices.

**Comment by bracketsberlin** [comment]
@mroseteo No, unfortunately not. We had to buy this component https://vapolia.eu/product/xamsvg

**Comment by Rogerdin13** [comment]
@mroseteo @bracketsberlin

There are more deviced affected by the SVGs not rendering (Samsung Galaxy A80, A71, S20, S10, S10+ to name a few).
I found the cause and solution here:  [https://github.com/luberda-molinet/FFImageLoading/issues/1526](https://github.com/luberda-molinet/FFImageLoading/issues/1526 )
solution: [https://github.com/luberda-molinet/FFImageLoading/issues/1526#issuecomment-773621747](https://github.com/luberda-molinet/FFImageLoading/issues/1526#issuecomment-773621747)

The SkiaSharp nuget needs an update to the newest version (v2.80.3 if im correct).
Atleast that fixed the Problem in my final project for me.
